app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -790,66 +816,106 @@
+        <item>MODE_12HR</item>
+        <item>MODE_24HR</item>
+    </string-array>
 
     <!-- App Setting: solstice tracking mode -->
     <string-array name="solsticeTrackingMode_display">                      <!-- one-to-one with solsticeTrackingMode_values -->
         <item>@string/trackingMode_closest</item>
         <item>@string/trackingMode_soonest</item>
     </string-array>
+    <string-array name="solsticeTrackingMode_values" translatable="false">  <!-- one-to-one with solsticeTrackingMode_display -->
+        <item>CLOSEST</item>
+        <item>SOONEST</item>
+    </string-array>
 
     <!-- App Setting: Date "on tap" action -->
     <string-array name="dateTapActions_display"> <!-- display values (one-to-one with dateTapActions_values) -->
         <item>Nada</item>
         <item>Inverter (hoje/amanhã)</item>
-        <item>Abrir app Calendário</item>
-        <item>Definir data</item>
+        <item>Mostrar app de calendário</item>
+        <item>Definir data customizada</item>
+    </string-array>
+    <string-array name="dateTapActions_values" translatable="false"> <!-- config values (one-to-one with dateTapActions_display) -->
+        <item>NOTHING</item>
+        <item>SWAP_CARD</item>
+        <item>SHOW_CALENDAR</item>
+        <item>CONFIG_DATE</item>
     </string-array>
 
     <!-- App Setting: Clock UI "on tap" action -->
     <string-array name="clockTapActions_display"> <!-- display values (one-to-one with clockTapActions_values) -->
         <item>Nada</item>
         <item>@string/configAction_setAlarm</item>
         <item>@string/configAction_setTimeZone</item>
-        <item>Mostrar próxima nota</item>
-        <item>Mostrar nota anterior</item>
+        <item>Exibir próxima nota</item>
+        <item>Exibir nota anterior</item>
+    </string-array>
+    <string-array name="clockTapActions_values" translatable="false">  <!-- config values (one-to-one with clockTapActions_display) -->
+        <item>NOTHING</item>
+        <item>ALARM</item>
+        <item>TIMEZONE</item>
+        <item>NEXT_NOTE</item>
+        <item>PREV_NOTE</item>
     </string-array>
 
     <!-- App Setting: Note UI "on tap" action -->
     <string-array name="noteTapActions_display">  <!-- display values (one-to-one with noteTapActions_values) -->
         <item>Nada</item>
         <item>Definir alarme para a nota</item>
-        <item>Mostrar próxima nota</item>
-        <item>Mostrar nota anterior</item>
+        <item>Exibir próxima nota</item>
+        <item>Exibir nota anterior</item>
+    </string-array>
+    <string-array name="noteTapActions_values" translatable="false">   <!-- config values (one-to-one with noteTapActions_display) -->
+        <item>NOTHING</item>
+        <item>ALARM</item>
+        <item>NEXT_NOTE</item>
+        <item>PREV_NOTE</item>
     </string-array>
 
     <!-- App Setting: time zone sort modes -->
-    <string name="timezone_sort_byOffset">by offset</string>  <!-- TODO -->
```

Comment:
I'm thinking these TODO probably still belong here. 

BTW, to see this particular text in action, goto `Set time zone`, select "User Defined", and then click on the "Time Zone" label (which should transition from a label to a button). This will bring up the sort menu.